Formula/gnunet.rb

```diff
@@ -35,9 +32,9 @@ class Gnunet < Formula
 
   def install
     ENV.deparallelize if OS.linux?
-    # Workaround for Xcode 14 ld until Makefile is fixed
-    # Ref: https://github.com/Homebrew/homebrew-core/pull/113789#issuecomment-1288125182
-    ENV.append_to_cflags "-fuse-ld=lld" if DevelopmentTools.clang_build_version >= 1400
+    # Workaround for LOGIN_NAME_MAX not defined on macOS
+    # Ref: https://bugs.gnunet.org/view.php?id=7557
+    ENV.append_to_cflags "-DLOGIN_NAME_MAX=255" if OS.mac?
```

Comment:
Probably shouldn't merge this unless can confirm the value across macOS systems. It is the value on my ARM Ventura machine:
```console
❯ getconf LOGIN_NAME_MAX
255
```

Perhaps using `sysconf(_SC_LOGIN_NAME_MAX)` would be a more accurate alternative.

Reply:
I can spin VM's for Big Sur and Monterey arm64 to test, but I don't own any Intel machines anymore.

Reply:
Monterey arm64 value is 255.